README.md

```diff
@@ -12,7 +12,7 @@ it by visiting our [HedgeDoc demo server][hedgedoc-demo].
 
 It is inspired by Hackpad, Etherpad and similar collaborative editors. This
 project originated with the team at [HackMD](https://hackmd.io) and now forked
-into its own organisation. [A longer writeup can be read in the history doc](docs/history.md).
+into its own organisation. [A longer writeup can be read in the history][history].
```

Comment:
Well this links not really to a longer write-up, anymore. Maybe try to use the same text I used in `docs/content/index.md`?